.github/stale.yaml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
This puts an obligation on us to react/resolve all issues in 2 months. I am not sure we can do that.